libraries/testAutomation/testAutomation/test_runner.py

```diff
@@ -247,18 +261,24 @@ def main() -> None:
     elasticsearch_cluster_chart_path = f"{helm_charts_base_path}/components/elasticsearchCluster"
     opensearch_cluster_chart_path = f"{helm_charts_base_path}/components/opensearchCluster"
 
-    # Currently utilizes a single test cluster environment, but should be expanded to allow a matrix of cases based
-    # on provided source and target version
-    ma_chart_values_path = "es-5-values.yaml"
-    es_5_6_values = (f"{helm_charts_base_path}/components/elasticsearchCluster/"
-                     f"environments/es-5-6-single-node-cluster.yaml")
-    os_2_19_values = (f"{helm_charts_base_path}/components/opensearchCluster/"
-                      f"environments/os-2-latest-single-node-cluster.yaml")
-    test_cluster_env = TestClusterEnvironment(source_version="ES_5.6",
-                                              source_helm_values_path=es_5_6_values,
+    if args.source_version == "ES_5.6":
+        ma_chart_values_path = "es-5-to-os-2-values.yaml"
+        source_values = (f"{helm_charts_base_path}/components/elasticsearchCluster/"
+                         f"environments/es-5-6-single-node-cluster.yaml")
+    elif args.source_version == "ES_8.18":
+        ma_chart_values_path = "es-8-to-os-2-values.yaml"
+        source_values = (f"{helm_charts_base_path}/components/elasticsearchCluster/"
+                         f"environments/es-8-latest-single-node-cluster.yaml")
```

Comment:
Can we use a convention that avoids hardcoding all these values, like we example the file to be named flavor 
'es', major '5', minor '6', and then just have the paths of these items follow those rules?